subsys/bluetooth/controller/ll_sw/ull_sched.c

```diff
@@ -50,129 +58,78 @@ static void win_offset_calc(struct ll_conn *conn_curr, uint8_t is_select,
 			    uint8_t *win_offset);
 #endif
 #endif /* CONFIG_BT_CTLR_CONN_PARAM_REQ */
+
+static uint8_t after_xxxx_slot_get(uint8_t user_id, uint32_t ticks_slot_abs,
```

Comment:
Curious name. Can you elaborate on the reasoning behind using `xxxx`?

Reply:
I could not settle for a name, earlier function was to find a time slot after master, but now the function is refactors to find a time slot after any scheduled ticker. Would `after_match_slot_get`  be a better function name?

Reply:
Possibly. I don't have a good answer for this function name :)

Regarding `master`, I assume that refers to `_mstr_`. Has that not been renamed in the core spec (if relevant to that) by the inclusive naming change? Looks like it has been overlooked.

Reply:
yes, it was overlooked, updated now.